src/components/Sidebar/Sidebar.jsx

```diff
@@ -169,6 +129,9 @@ Sidebar.propTypes = {
     isStickyHeader: PropTypes.bool,
     isStickyFooter: PropTypes.bool,
     onLogoClick: PropTypes.func.isRequired,
+    isLeftDrawerOpen: PropTypes.bool.isRequired,
+    isRightDrawerOpen: PropTypes.bool.isRequired,
+    handleDrawerStateChange: PropTypes.func.isRequired,
```

Comment:
I don't think `handleDrawerStateChange`, `isLeftDrawerOpen` and `isRightDrawerOpen` should be required because not all consumers of the Sidebar component will be implementing the a drawer with the Sidebar.